python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -53,7 +60,47 @@ def _is_fastai_model(model):
     return str(type(model)).endswith(FASTAI_MODEL_SUFFIX)
 
 
-def _wrap_image_model(model, examples, model_task, is_function):
+def _filter_score(orig_prediction: dict, score_thresh: float = 0.5):
+    """Filter out predictions with confidence scores < score_thresh.
+
+    :param orig_prediction: Original model prediction
+    :type orig_prediction: dict
+    :param score_thresh: Score threshold to filter by
+    :type score_thresh: float
+    :return: Model predictions filtered out by score_thresh
+    :rtype: dict
+    """
+    keep = orig_prediction['scores'] > score_thresh
+
+    filter_prediction = orig_prediction
+    filter_prediction['boxes'] = filter_prediction['boxes'][keep]
```

Comment:
nit: move repeating strings to top of file as constants, including:

```
'boxes'
'scores'
'labels'
```

you can set them at top of file as:

```
BOXES = 'boxes'
LABELS = 'labels'
SCORS = 'scores'
```